src/SDK/SDK.php

```diff
@@ -69,6 +69,144 @@ class SDK
         'examples' => '',
         'test' => 'false'
     ];
+    private array $errorTypes = [
```

Comment:
Where did this come from and why is it needed? 🧐

Reply:
We are using this data, taken from the spec to generate the tests for different SDKs